noxfile.py

```diff
@@ -417,3 +420,32 @@ def create_parser() -> ArgumentParser:
         *args,
         external=True,
     )
+
+
+@nox.session(python=False, name="clean-docs")
+def clean(session: Session) -> None:
+    """Remove all documentation artifacts"""
+    if DOC_BUILD.exists():
+        rmtree(DOC_BUILD.resolve())
+        session.log(f"Removed {DOC_BUILD}")
+
+
+@nox.session(python=False, name="build-docs")
+def build(session: Session) -> None:
+    """Build the documentation"""
+    session.run(
```

Comment:
Do we also want to generate API doc or doesn't make this sense for this project?

Reply:
Currently we don't, was thinking of doing it but my thought was one would rather check the `sqla` documentation. The `sqlalchemy-exasol` code, is/should never be invoked directly by the `user`.